wiki/Beatmap_Discussion/en.md

```diff
@@ -38,11 +38,11 @@ Clicking on `Modding/Discussion` on the beatmap's page will take you to the page
 
 Changes the game mode between osu!standard, osu!taiko, osu!catch, and osu!mania. You can only select each game mode if the beatmapset has a difficulty with said game mode.
 
-## Beatmap page / Difficulty selection
+### Beatmap page / Difficulty selection
 
 ![](img/difficulty-menu.jpg "Difficulty menu")
 
-Clicking on the Artist/Title of the beatmap will take you back to the Beatmap Information page. You can select a difficulty to mod using the dropdown menu. **Make sure you are on the correct difficulty before submitting mods to the beatmap!**
+Clicking on the Artist/Title of the beatmap will take you back to the Beatmap Information page. You can select a difficulty to mod using the dropdown menu. The numbers that appear besides the difficulty name are the number of pending discussions it currently has to resolve. **Make sure you are on the correct difficulty before submitting mods to the beatmap!**
```

Comment:
The number that appear
it currently has to resolve -> which currently need to be resolved?

Reply:
is this okay?
``The number that appears besides the difficulty name are the number of pending discussions it currently has that needs to be resolved.``

Reply:
the number **are**?

Reply:
can you explain further on 
> the number **are**?

Reply:
The number that appears besides the difficulty name is the number of pending discussions which have to get resolved.

You wrote "The number are"

Reply:
I have yet to understand what you mean, I am pretty sure that that you spoted was even there. 😕

Reply:
```is this okay?
The number that appears besides the difficulty name are the number of pending discussions it currently has that needs to be resolved.```
you wrote "The number (...) are"